src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import './CommentInfo.scss';
+
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const {
+    name,
+    email,
+    body,
+  } = comment;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">{name}</strong>
+
+        {' by '}
```

Comment:
why do you insert it as js code?
```suggestion
        by
```

Reply:
Because I wanted to save spaces before and after 'by'. I don't know, how to do it(

Reply:
![Screenshot_45](https://user-images.githubusercontent.com/110488158/202483221-9600025d-ed5d-4572-8856-0c0fdaebf80c.png)
![Screenshot_46](https://user-images.githubusercontent.com/110488158/202483224-500e480e-aa77-41c1-8847-d1f117d1932f.png)
To add spaces in the next element?

Reply:
![Screenshot_48](https://user-images.githubusercontent.com/110488158/202483873-509d81db-f988-4cee-b5f9-b08c345297d3.png)
![Screenshot_47](https://user-images.githubusercontent.com/110488158/202483880-eadd8311-a742-472d-821b-b286f15a5bdf.png)
To add spaces in the next element?